linopy/solvers.py

```diff
@@ -947,6 +947,8 @@ def run_mosek(
 
             if log_fn is not None:
                 m.linkfiletostream(mosek.streamtype.log, log_fn, 0)
+            else:
+                m.set_Stream(mosek.streamtype.log, lambda x: sys.stdout.write(x))
```

Comment:
```suggestion
                m.set_Stream(mosek.streamtype.log, sys.stdout.write)
```
This likely also works